integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/table/CarbonShowTablesCommand.scala

```diff
@@ -44,39 +44,15 @@ private[sql] case class CarbonShowTablesCommand ( databaseName: Option[String],
     val db = databaseName.getOrElse(catalog.getCurrentDatabase)
     var tables =
       tableIdentifierPattern.map(catalog.listTables(db, _)).getOrElse(catalog.listTables(db))
-    tables = filterDataMaps(tables, sparkSession)
+    val externalCatalog = sparkSession.sharedState.externalCatalog
```

Comment:
Update the comment "filterDataMaps Method is to Filter the Table." as filterDataMaps is removed